cranelift/codegen/meta/src/isa/x86.rs

```diff
@@ -17,19 +17,22 @@ fn define_settings(shared: &SettingGroup) -> SettingGroup {
         "has_sse3",
         "Has support for SSE3.",
         "SSE3: CPUID.01H:ECX.SSE3[bit 0]",
-        false,
+        // Needed for default `enable_simd` setting.
+        true,
```

Comment:
This means explicitly using the baseline preset will now incorrectly include these flags, right?

Reply:
Will include those flags, yes; "incorrectly" is a matter of interpretation, as we effectively need them at the moment to support our default `enable_simd` setting. Otherwise a default config will not work with default Wasmtime settings.

Reply:
If I enable the baseline preset and not enable_simd, then these features shouldn't be enabled as it won't run on a cpu only implementing the x86_64 baseline. Maybe add them as dependency of enable_simd or enable them in wasmtime?

Reply:
We could do it that way as well; happy to take a PR if you want to refactor in that direction. From my point of view, this should be relatively temporary anyway, as we should eventually (#3810) be able to do SIMD without any optional extensions.